Error while build

I got a typescript error while running ./build.sh

test/item-list-test.ts(152,5): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(153,5): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(549,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(550,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(561,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(562,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(573,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(574,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(622,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(623,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(627,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(628,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(636,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(637,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(641,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(642,7): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(661,9): error TS2554: Expected 2 arguments, but got 1.
test/item-list-test.ts(683,9): error TS2554: Expected 2 arguments, but got 1.
test/notebook-preview-test.ts(82,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(97,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(112,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(125,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(138,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(151,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.
test/notebook-preview-test.ts(162,14): error TS2345: Argument of type '{}' is not assignable to parameter of type 'boolean'.

--- a/sources/web/datalab/polymer/test/test-utils.ts
+++ b/sources/web/datalab/polymer/test/test-utils.ts
@@ -16,7 +16,6 @@
 /// <reference path="../node_modules/@types/chai/index.d.ts" />
 /// <reference path="../node_modules/@types/sinon/index.d.ts" />
 
-declare function assert(condition: boolean, message: string): null;
 declare function fixture(element: string): any;
 
 class MockFile extends DatalabFile {

